server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -139,6 +141,7 @@ public static void download(TranslogTransferManager translogTransferManager, Pat
                 location.resolve(Translog.CHECKPOINT_FILE_NAME)
             );
         }
+        logger.info("Downloaded translog files from remote for shard {} ", translogTransferManager.getShardId());
```

Comment:
`translogTransferManager.downloadTranslog` already has a log line for downloading files of each generation. If we add shardId to that log, will it help or do we still need these logs?

Reply:
Yes, that is just for `downloading` and not when it completes . So if we want to measure end to end time of download , it would make our job harder .

Reply:
Should we have the log level as trace or debug?

Reply:
This is used very sparingly at the time of restore/failover/relocation . Hence info makes sense .